docs/competition/teams/role-descriptions/reception.md

```diff
@@ -56,11 +47,9 @@ Volunteers will be turning up throughout the competition, it's important to make
 
 You should have the following items present for your duties. If you do not, please let the competition event coordinator know as soon as possible.
 
-* Spare Media Consent Forms (MCFs) to allow visitors to enter.
-* Pens to fill in forms.
-* Wristbands to give to entrants.
+* Pens.
 * Competitor booklets to give to teams.
-* This documentation and flow chart.
+* This documentation.
 * Sign-in/out sheet.
```

Comment:
Is this the same sheet for teams & volunteers or are they separate? Maybe clarify this either way?

Reply:
These have been sepereated